packages/ui/src/theme/css/component/_fieldControl.scss

```diff
@@ -1,4 +1,4 @@
-%amplify-field-control {
```

Comment:
`%` and `@mixin` fill very similar roles here in that they let us add this block of styling to other rules. `%` is a placeholder selector and just means that this block isn't output in the final css on its own and is only used to extend other sass rules. Because we aren't passing in any variables here we would normally use `%` instead of `@mixin` but the `&--{modifier}` rules weren't being compiled correctly with `%` so that is why this was switched to `@mixin`

Reply:
That's weird.`%` and `@extend` is supposed to be used with BEM - https://sass-lang.com/documentation/at-rules/extend